docker/README.md

```diff
@@ -43,6 +43,32 @@ Once this is done you should run the following command to build `aarch64` static
 ./docker-static-build.sh aarch64
 ```
 
+### Packages from statically linked binaries
+
+It's possible to create packages (ubuntu/debian & fedora) using already
+built binaries. This means that the build step of the package will be reduced
+to copying the binaries instead of building again from scratch.
+
+Using Ubuntu/Debian and assuming all Tezos binaries are located in the `out` directory:
+```sh
+# DEB_BUILD_OPTIONS=nostrip is needed if the provided binaries have already
+# been stripped of any debug symbols.
+
+docker build -t tezos-ubuntu -f docker/package/Dockerfile-ubuntu .
+docker run -it -e TEZOS_VERSION=v13.0 \
```

Comment:
Tried following these instructions, got the following:
```
docker run -it -e TEZOS_VERSION=v13.0 \
               -e DEB_BUILD_OPTIONS=nostrip \
               -v `pwd`:/tezos-packaging tezos-ubuntu \
               --os ubuntu --type binary --binaries-dir ./out            
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/tezos-packaging/docker/package/package_generator.py", line 122, in <module>
    build_ubuntu_package(
  File "/tezos-packaging/docker/package/ubuntu.py", line 28, in build_ubuntu_package
    pkg.fetch_sources(dir, binaries_dir)
  File "/tezos-packaging/docker/package/model.py", line 228, in fetch_sources
    os.makedirs(out_dir)
  File "/usr/lib/python3.8/os.py", line 223, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: 'tezos-client-13.0'
```
Am I missing something?